wildfirepy/net/usgs/viirs.py

```diff
@@ -121,3 +121,13 @@ class VIIRSBurntAreaDownloader(Viirs):
     """
     def __init__(self):
         super().__init__(product="VNP03MODLL")
+
+
+class VIIRSBurntAreaDownloader1(Viirs):
```

Comment:
```suggestion
class VIIRSBurntAreaData(Viirs):
```
We need to edit the name of the other class as well but that's a separate issue.